scripts/SeafoamIslandsB4F.asm

```diff
@@ -137,6 +137,10 @@ SeafoamIslandsB4F_TextPointers:
 	dw SeafoamIslands5Text4
 	dw SeafoamIslands5Text5
 
+; Articuno is object 3, but its event flag is bit 2.
+; This is not a problem because its sight range is 0, and
+; trainer headers were not stored by ExecuteCurMapScriptInTable.
```

Comment:
Is the 3rd line of this comment useful?

Reply:
It's relevant because without `ExecuteCurMapScriptInTable` being called, no `SeafoamIslandsB4FTrainerHeaders` label needs to exist at all, let alone matching the object IDs and bits.